apps/camera/js/controllers/controls.js

```diff
@@ -27,10 +27,32 @@ function ControlsController(app) {
   this.activity = app.activity;
   this.createView();
   this.bindEvents();
+  this.l10nGet = app.l10nGet;
+  this.updateSwitchLabels();
   debug('initialized');
 }
 
 /**
+ * Updates app `batteryStatus` and
+ * manages battery notifications.
+ *
+ * @private
+ */
+ControlsController.prototype.updateSwitchLabels = function () {
+  // We need the app to be first localized
+  // before setting proper aria-Labels
+  if (!this.app.localized()) {
+    this.app.on('localized', this.updateSwitchLabels);
```

Comment:
If the app is already localized, this listener never gets added. This would be bad because if the user changes locale, it won't get updated.